tests/containers/install_updates.pm

```diff
@@ -17,10 +17,13 @@ sub run {
     my ($self) = @_;
     my $update_timeout = 2400;
     my ($version, $sp, $host_distri) = get_os_release;
+    # Check for bsc1232902, affecting the last 15-SP7 ISO, /etc/os-release paameters.
+    my $is_bsc_1232902 = (is_sle("=15-sp7") and $host_distri =~ /^sle$/);
 
     # Update the system to get the latest released state of the hosts.
     # Check routing table is well configured
-    if ($host_distri =~ /sles|opensuse/) {
+    if ($host_distri =~ /sles|opensuse/ or ($is_bsc_1232902)) {
+        record_soft_failure("bsc#1232902, [Build GM] openQA SLES test fails in install_updates for unexpected os-release ID") if ($is_bsc_1232902);
```

Comment:
I think it's better for the readability to have a separate `elsif` block with the condition for bsc#1232902 than mixing it in here. WDYT? :thinking:

Reply:
I discarded this option because we shall duplicate/repeat the same code block for each `OR` test-expr.

Reply:
Instead, using the var.name like our `is_xxx()` test routines, readability is achieved and also code simplicity.